CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file. See [Keep a
 CHANGELOG](http://keepachangelog.com/) for how to update this file. This project
 adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [0.5.2] - 2017-10-18
```

Comment:
Let's keep the unreleased tag for now--we'll do a separate commit on master when we release (which will then be tagged with the version number).